src/flashbar/__integ__/collapsible.test.ts

```diff
@@ -2,13 +2,14 @@
 // SPDX-License-Identifier: Apache-2.0
 import { setupTest } from './pages/interactive-page';
 import { FOCUS_THROTTLE_DELAY } from '../utils';
+import { wait } from '../../internal/utils/promises';
 
 describe('Collapsible Flashbar', () => {
   describe('Keyboard navigation', () => {
     it(
       'can expand and collapse stacked notifications with the keyboard',
       setupTest(async page => {
-        await page.toggleCollapsibleFeature();
+        await page.toggleStackingFeature();
```

Comment:
Updated according to new naming convention. There a still places using the term "collapsible", but not sure I want to change it everywhere. The feature is about "stacking", and when using it the notifications become "stacked", but it might still make sense to say that the Flashbar becomes "collapsible". In any case further changes are out of scope here